lib/recurly/hosted-field.js

```diff
@@ -82,6 +87,16 @@ export class HostedField extends Emitter {
     });
   }
 
+  bindTabTrap () {
+    this.tabTrapInput.addEventListener('focus',(e) => {
+      e.preventDefault()
+      e.stopPropagation()
+      // Firefox demands a blur or else the focus doesn' work...
+      if(typeof InstallTrigger !== 'undefined') e.currentTarget.blur()
```

Comment:
generic FF detection for FF 1.0+. It's the method they use for installing Addons or something.